go/install/install_osx.go

```diff
@@ -161,6 +161,7 @@ func ListServices(g *libkb.GlobalContext) (*keybase1.ServicesStatus, error) {
 func DefaultLaunchdEnvVars(g *libkb.GlobalContext, label string) []launchd.EnvVar {
 	return []launchd.EnvVar{
 		launchd.NewEnvVar("KEYBASE_LABEL", label),
+		launchd.NewEnvVar("KEYBASE_SERVICE_TYPE", "launchd"), // This was added 03/22/16
```

Comment:
Fixed check in service/main.

Good point on args vs env vars... Env vars were easier to add.. That said...
I removed KEYBASE_RUNTIME_DIR and set kbfs flag -runtime-dir. (This env var was ignored by main service).
I also removed KEYBASE_LOG_FORMAT since thats default now that we're using log files.

We could remove KEYBASE_LABEL and KEYBASE_SERVICE_TYPE but would take more changes to add cli flags. I can do that if we want? Or should I do that in a separate PR?